rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy

```diff
@@ -642,7 +655,10 @@ Authorization required: `read` for project resource
 *Since API version 26*: add the project `label` to the response
 
 *Since API version 33*: add the project `created` date to the response. This is based on the creation of the 
-`project.properties` file in the file system or in the DB storage.''',
+`project.properties` file in the file system or in the DB storage.
+
+*Since API version 46*: add the project `metadata` to the response. To retrieve this information, use the query param meta
```

Comment:
```suggestion
*Since API version 47*: add the project `metadata` to the response. To retrieve this information, use the query param `meta`
```

Reply:
also, there should be a new `@Parameter` for the `meta` param for the OpenAPI spec that specifies the parameter details, see: 

annotated method param:
https://github.com/rundeck/rundeck/blob/4c390a5e2bbd48abdab9ff8f449a4ce973afccfc/rundeckapp/grails-app/controllers/rundeck/controllers/ScheduledExecutionController.groovy#L6171-L6176

or within the `@Operation`: https://github.com/rundeck/rundeck/blob/f8237b208edef4752f1907a170ea54d213cdea1f/rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy#L962-L969